cmd/armada/destroy/cluster/destroycluster.go

```diff
@@ -0,0 +1,62 @@
+package cluster
+
+import (
+	"io/ioutil"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/armada/pkg/cluster"
+	"github.com/submariner-io/armada/pkg/defaults"
+	kind "sigs.k8s.io/kind/pkg/cluster"
+)
+
+// DestroyClusterFlagpole is a list of cli flags for destroy clusters command
+type DestroyClusterFlagpole struct {
+	Clusters []string
+}
+
+// DestroyClustersCommand returns a new cobra.Command under destroy command for armada
+func DestroyClustersCommand(provider *kind.Provider) *cobra.Command {
+	flags := &DestroyClusterFlagpole{}
+	cmd := &cobra.Command{
+		Args:  cobra.NoArgs,
+		Use:   "clusters",
+		Short: "Destroy clusters",
+		Long:  "Destroys clusters",
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			var targetClusters []string
+			if len(flags.Clusters) > 0 {
+				targetClusters = append(targetClusters, flags.Clusters...)
+			} else {
+				configFiles, err := ioutil.ReadDir(defaults.KindConfigDir)
+				if err != nil {
+					log.Fatal(err)
+				}
+				for _, configFile := range configFiles {
+					clName := strings.FieldsFunc(configFile.Name(), func(r rune) bool { return strings.ContainsRune(" -.", r) })[2]
+					targetClusters = append(targetClusters, clName)
```

Comment:
I'm guessing that this is something you do in multiple places and not just here. If that's true then make a function.